app/containers/TodoApp/TodoApp.js

```diff
@@ -0,0 +1,58 @@
+/* @flow */
+
+import React from 'react';
+import Relay from 'react-relay';
+import TodoForm from './components/TodoForm';
+import TodoList from './components/TodoList';
+import AddTodoMutation from './mutations/AddTodoMutation';
+
+type Props = {
+  user: {
+    todos: any,
+  },
+  relay: {
+    commitUpdate: Function,
+  },
+};
+
+/**
+ * Container for todo application
+ * Parent component: N/A
+ * Child components: TodoList, TodoForm
+ */
+class TodoApp extends React.Component {
+
+  props: Props;
+
+  // addTodo function passed to the TodoForm
```

Comment:
can we have standardized function descriptions including parameters and return values if applicable?
```
/**
 * [function description]
 * @param {[type]} [description]
 * @return {[type]} [description]
 */
```